src/stumpy_png/datastream.cr

```diff
@@ -16,7 +16,9 @@ module StumpyPNG
     end
 
     def self.read(io : IO)
-      raise "Not a png file" unless io.read_bytes(UInt64, IO::ByteFormat::BigEndian) == StumpyPNG::HEADER
```

Comment:
To stay consistent with the `!=` in the CRC check, this could be rewritten with `if` and `!=`

Reply:
I don't see a reason for consistency here. Also, it was like that before my changes.